packages/api/cms-api/src/auth/guards/comet.guard.ts

```diff
@@ -26,7 +26,7 @@ export function createCometAuthGuard(type?: string | string[]): Type<IAuthGuard>
             if (user) {
                 return user;
             }
-            throw new HttpException("UserNotAuthenticated", 200);
+            throw new HttpException("UNAUTHENTICATED", 200);
```

Comment:
Why do we throw a 200 error here? For GraphQL? This guard is also used for REST operations, should we change that (probably in next)?